structured_params.gemspec

```
@@ -9,8 +9,8 @@ Gem::Specification.new do |spec|
   spec.authors = ['Mizuki Yamamoto']
   spec.email = ['mizuki-y@syati.info']
 
-  spec.summary = 'Type-safe parameter validation and form objects for Rails.'
-  spec.description = ''
+  spec.summary = 'Typed parameter objects and form objects for Rails.'
+  spec.description = 'StructuredParams provides typed parameter objects for Rails APIs and forms, with nested object and array support, Strong Parameters integration, and raw-input validation via validates_raw.'
```

RuboCop (1.88.0)

[convention] Line is too long. [211/120]

(Layout/LineLength)